src/hooks/useAuth.ts

```
@@ -0,0 +1,327 @@
+  const validateField = useCallback((name: string, value: string) => {
+    let error = '';
+    
+    switch (name) {
+      case 'email':
+        const emailValidation = AuthValidator.validateEmail(value);
+        if (!emailValidation.isValid) {
+          error = emailValidation.errors[0];
+        }
+        break;
+      case 'password':
+        const passwordValidation = AuthValidator.validatePassword(value);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)